### PR TITLE
mention docker image is useful to windows users

### PR DESCRIPTION
**Pull request by mikep01**
Fixes #\<GitHub-issue-number\>.

Make sure all boxes are checked (add x inside the brackets) when you submit your contribution, remove this sentence before doing so.

- [ ] This PR is compliant with the [contributing guidelines](https://github.com/nodemcu/nodemcu-firmware/blob/dev/CONTRIBUTING.md) (if not, please describe why).
- [ ] I have thoroughly tested my contribution.
- [ ] The code changes are reflected in the documentation at `docs/en/*`.

\<Description of and rational behind this PR\>

Committers supporting this PR: leave blank

### CODE REVIEW

File: docs/en/build.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Note that the *default configuration in the C header files* (`user_config.h`, `u
 NodeMCU "application developers" just need a ready-made firmware. There's a [cloud build service](http://nodemcu-build.com/) with a nice UI and configuration options for them.
 
 ## Docker Image
-Occasional NodeMCU firmware hackers don't need full control over the complete tool chain. They might not want to setup a Linux VM with the build environment. Docker to the rescue. Give [Docker NodeMCU build](https://hub.docker.com/r/marcelstoer/nodemcu-build/) a try.
+Occasional NodeMCU firmware hackers don't need full control over the complete tool chain. They might not want to setup a Linux VM with the build environment. Docker to the rescue. Give [Docker NodeMCU build](https://hub.docker.com/r/marcelstoer/nodemcu-build/) a try.  Very useful for Windows users that need custom builds.
```
Comment (marcelstoer) [inline docs/en/build.md]: Maybe you should include "in particular" in that sentence. Linux users can use the full toolchain as described below and Mac users have a chance to adjust the toolchain to their OS (very similar to Linux).

**Comment by jmattsson** [comment]
This is against `master`; needs to be redone against `dev`. PR template not filled in.

**Comment by mikep01** [comment]
Sorry about that, I'm new to the process and was just trying to help.  I followed the link " Edit on GitHub" on that published paged, found the file and made the change, not knowing I need to select a different branch.  Perhaps that link should point to a process description first?

**Comment by marcelstoer** [comment]
No worries Mike, we all started somewhere... I updated the PR template GitHub shows just recently to make that more explicit. It'll become active with the next release.

**Pull request closed without merging**